frontend/app/components/input-label.tsx

```diff
@@ -16,9 +16,13 @@ export const InputLabel = (props: InputLabelProps) => {
   return (
     <label className={clsx(required && 'required', className)} data-testid="input-label" {...restLabelProps}>
       <span className="field-name">{children}</span>
-      {required && <strong className="required">&#32;({t('input-label.required')})</strong>}
+      {required && (
+        <strong className="required" data-testid="input-label-required">
+          &#32;({t('input-label.required')})
```

Comment:
Why not just put a small left margin on the element to emulate a space?

Reply:
Marking as `/wontfix` ... we can revisit later.

Reply:
Without it the label text will be read like "Email address(Required)". 🤷